packages/cosmic-swingset/lib/ag-solo/vats/types.js

```diff
@@ -10,20 +10,26 @@
 
 /**
  * @typedef {Object} NameHub
- * @property {(...path: Array<unknown>) => Promise<unknown>} lookup Look up a
+ * @property {(...path: Array<string>) => Promise<any>} lookup Look up a
```

Comment:
Please justify the use of `any`.

What I've done in similar cases is make `lookup` polymorphic and add an `example` argument that supplies the type.
https://github.com/dckc/finquick/blob/master/ofxies/server/budget.js#L307
https://github.com/dckc/finquick/blob/master/ofxies/server/budget.js#L116
https://github.com/dckc/finquick/blob/master/ofxies/server/budget.js#L82

Reply:
never mind. the budget.js approach isn't really typesafe anyway